server_admin/topics/users/proc-completing-registration-form.adoc

```diff
@@ -1,17 +1,20 @@
-[id="_{context}"]
-= Completing the egistration form 
+[id="completing-registration-form_{context}"]
+= Completing the registration form 
 
-.Procedure
+[role="_abstract"]
+The registration form can be customized by removing or adding additional fields that must be completed by a user.
```

Comment:
Better to move this to just before the link to the Developers guide under Additional resources.

Reply:
Of course, you need an introductory sentence here.  Perhaps "As a new user, you complete a form with your profile information."

Reply:
why does a user need to complete the form? Are they denied access if they do not? Will they run into the disruption of service if they do not complete the form?

Reply:
I haven't actually tried it. So I don't know, but they obviously need a password.